client/header/activity-panel/style.scss

```diff
@@ -44,6 +44,15 @@
 		width: 18px;
 		height: 18px;
 
+		// custom progress icon, requires specific coloring
+		&.setup-progress {
```

Comment:
Setup progress is different from all the other icons, its a 2 toned SVG rather than all the other tab icons which are dashicons.